packaging/nnstreamer.spec

```diff
@@ -220,7 +220,9 @@ BuildRequires:  pkgconfig(amlogic-vsi-npu-sdk)
 %endif
 
 %if 0%{?grpc_support}
-BuildRequires:  grpc-devel
+BuildRequires:	pkgconfig(re2)
+BuildRequires:	pkgconfig(libcares)
+BuildRequires:	grpc-devel
```

Comment:
If building packages with grpc-devel requires re2 and libcares, grpc package should declare that its devel packages requires the two. Or is it because of our internal needs?

Reply:
Reference https://build.opensuse.org/package/view_file/devel:tools/grpc/grpc.spec?expand=1

Reply:
Yes, I know and totally agree with you. But, this was a request from the Tizen gRPC maintainer (by email).
So, then we just let him resolve dependency issues by himself.